### DISCUSSION TITLE
DOM differ

### DISCUSSION THREAD

**Discussion by Wulfheart** [alpinejs/alpine#730] [4. General]
In the most recent episode of "No Plans to Merge" @calebporzio mentions that a DOM differ for alpine/livewire would be beneficial. 
I'd be happy to contribute a webassembly version of a dom-differ. What are the requirements? Let's discuss!

_Note: I'm going to post this at the alpine and the livewire repo as I'm not 100% sure where it belongs to definitely._

**Comment by HugoDF** [comment]
There's not much DOM diffing being done in Alpine.js at the moment, but I think Livewire does more of that.

**Reply by Wulfheart** [reply]
In this case I’m going to discuss at livewire.
